packages/babel-parser/test/fixtures/flow/type-annotations/140/input.js

```diff
@@ -0,0 +1,4 @@
+class A {
```

Comment:
I know we do a really bad job of it, but can we give this folder a name? Maybe `class-private-property` or something?